db/keyval/plugin/skeleton.go

```diff
@@ -48,7 +49,11 @@ func (plugin *Skeleton) Init() (err error) {
 // AfterInit is called once all plugin have been initialized. The connection to datastore
 // is established in this phase.
 func (plugin *Skeleton) AfterInit() (err error) {
-	plugin.conn, err = plugin.connect(logroot.Logger()) // TODO: use injected logger
+	l, err := plugin.Lg.NewLogger(plugin.name)
```

Comment:
I think according to the https://github.com/golang/go/wiki/CodeReviewComments#variable-names actual name should be fine ...but I can rename it